vaadin-rich-text-editor-flow-integration-tests/pom.xml

```diff
@@ -169,13 +163,6 @@
             from GitHub -->
         <!--Do not use with production mode -->
         <profile>
```

Comment:
The rest of the profile should be removed, along with the comment above.

Reply:
Done

Reply:
Done